Philips.CodeAnalysis.Common/Helper.cs

```diff
@@ -183,12 +181,11 @@ public static bool HasAttribute(AttributeListSyntax attributes, SyntaxNodeAnalys
 		public static bool IsAttribute(AttributeSyntax attribute, SyntaxNodeAnalysisContext context, string name, string fullName, out Location location, out AttributeArgumentSyntax argument)
 		{
 			location = null;
-			argument = default(AttributeArgumentSyntax);
+			argument = default;
 
 			if (attribute.Name.ToString().Contains(name))
 			{
-				IMethodSymbol memberSymbol = context.SemanticModel.GetSymbolInfo(attribute).Symbol as IMethodSymbol;
-				if (memberSymbol != null && memberSymbol.ToString().StartsWith(fullName))
+				if (context.SemanticModel.GetSymbolInfo(attribute).Symbol is IMethodSymbol memberSymbol && memberSymbol.ToString().StartsWith(fullName))
```

Comment:
Code seems familiar, see Line 156

Reply:
Agreed, but I don't want to touch the logic in this large PR.